### ISSUE TITLE
exploring cloud9.io with a view to using it for a hackathon / provisioning / teaching

### ISSUE THREAD

**Issue by christopherreay** [metacurrency/holochain#261]
https://docs.c9.io/docs/running-your-own-ssh-workspace

**Comment by christopherreay** [comment]
Hey, @artbrock , any change you could try to create an account on cloud9? The free one doesnt do "ssh projects" and by the looks of the documentation, they dont accept non-us debit cards

**Comment by christopherreay** [comment]
I wonder if I could just install some kind of collaborative code editor on the server :)

**Comment by artbrock** [comment]
I can create a cloud9 account. I'll get back to you with the info.

**Comment by christopherreay** [comment]
I discovered that the interface behaves as if one has a free account, even after invoicing, until a member is added to an educational team. 

I have been in dialogue with "Brady"@cloud9 about it.

**Comment by christopherreay** [comment]
Nope, it doesnt. They have already billed me, and they dont seem to have a clue what is going on. Ill wait for "Brady" to get back to me again

**Comment by artbrock** [comment]
Well... if you get a workspace set up that you want to share. I'm signed up on c9 as artbrock

**Comment by christopherreay** [comment]
Team subscription and premium account are independent, and being member of a team still allows to have individual premium account. Please see https://community.c9.io/t/education-plan-faqs/9878 and https://community.c9.io/t/cloud9-for-teams-faq/4248 for more detailed description.
Educational teams do not include ssh workspaces, or extra resources, they only allow to invite students without credit card, and manage workspaces created by the students.

Best,
Harutyun

**Comment by christopherreay** [comment]
So this is the response from Cloud9:


Sent to christopher reay and Jay DayZee

Hi Christopher,

Thank you for reaching out about this. I'm afraid we don't have trial accounts, but if you give Premium a try for 5 days and find out that it doesn't work for your use case we'll be happy to cancel the subscription and refund the full amount. Be sure to reply to this thread, though, so we'll be able to see the history and apply the refund!

Kind regards,
Dan
Cloud9 IDE
